deploy/gatekeeper.yaml

```diff
@@ -281,7 +281,7 @@ spec:
     spec:
       containers:
       - args:
-        - --auditInterval=30
+        - --audit-interval=30
```

Comment:
We should update these flags in this yaml and the one in the helm chart _after_ we cut a new release. Otherwise this deployment will fail with the old image tag.

Reply:
Ignore this. Just seeing `checkDeprecatedFlags` below

Reply:
+1

Reply:
Good catch -- removed those for now